wayback/_utils.py

```diff
@@ -226,7 +226,9 @@ def rate_limited(calls_per_second=2, group='default'):
             minimum_wait = 1.0 / calls_per_second
             current_time = time.time()
             if current_time - last_call < minimum_wait:
-                time.sleep(minimum_wait - (current_time - last_call))
+                secs = minimum_wait - (current_time - last_call)
+                logger.debug("sleeping %s", secs)
```

Comment:
I think it would be helpful if this message specified that the cause is the rate limiter, e.g:

```suggestion
                logger.debug("Hit rate limit, sleeping for %s seconds", secs)
```